internal/scheduler/scheduler.go

```diff
@@ -167,22 +181,22 @@ func (s *Scheduler) start() {
 
 			repo, err := s.jobRepoFn()
 			if err != nil {
-				s.logger.Error("error creating job repo", "error", err)
+				s.logger.Debug("error creating job repo", "error", err)
```

Comment:
Why are we logging the errors at Debug level?

Reply:
ty, I made all the logs `Error` because I was not sure how to set debug level in unit tests so I could see the logs to make sure it was all working.  I must have made a grep error :/